brave-intl/publishers
# Task/3504 wordpress

## Pull Request Name

Remove wordpress detection for new channels.

#### Features

- Removed legacy wordpress detection

#### How To Test

1. Add a new channel
2. Enter "wordpress.com" as the domain
3. Continue the adding workflow.  Wordpress.com will now require a DNS entry for verification and will no longer display the custom "Wordpress Detected" view.

#### Pull Request Checklist
